bundles/org.openhab.binding.amplipi/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -83,6 +88,11 @@
 				<label>Group ID</label>
 				<description>The ID of the group</description>
 			</parameter>
+			<parameter name="volumeDelta" type="integer" min="1" max="10">
```

Comment:
Could we make this an "advanced" parameter?

Reply:
Done - wasn't aware this was an option - it's been a while since my last OH contrib!!